internal/charmstore/addentity.go

```diff
@@ -515,13 +513,12 @@ func (s *Store) addEntity(entity *mongodoc.Entity) (err error) {
 		Write: perms,
 	}
 	baseEntity := &mongodoc.BaseEntity{
-		URL:             entity.BaseURL,
-		User:            entity.User,
-		Name:            entity.Name,
-		Public:          false,
-		ACLs:            acls,
-		DevelopmentACLs: acls,
```

Comment:
I've only removed filling them out, they are still in the document. If I read the design correctly they will be modified slightly anyway.